testsuite/pytests/test_compartmental_model.py

```diff
@@ -962,6 +966,145 @@ def test_add_assign(self):
         cm2.receptors += [{"comp_idx": 1, "receptor_type": "AMPA"}, {"comp_idx": 2, "receptor_type": "GABA"}]
         self.assertEqual(len(list(cm2.receptors)), 3)
 
+    def test_custom_v_init(self):
+        sp0 = {'C_m': 1.00, 'g_C': 0.00, 'g_L': 0.100, 'e_L': -60.0}
+        dp0 = {'C_m': 0.10, 'g_C': 0.10, 'g_L': 0.010, 'e_L': -60.0}
+        sp1 = {'C_m': 1.00, 'g_C': 0.00, 'g_L': 0.100, 'e_L': -75.0}
+        dp1 = {'C_m': 0.10, 'g_C': 0.10, 'g_L': 0.010, 'e_L': -80.0}
+        sp2 = {'C_m': 1.00, 'g_C': 0.00, 'g_L': 0.100, 'e_L': -70.0}
+        dp2 = {'C_m': 0.10, 'g_C': 0.10, 'g_L': 0.010, 'e_L': -70.0}
+        v_init = -60.
```

Comment:
Maybe also an opportunity to change your test to the new modern `pytest` and would be better to put those lines inside a **pytest fixture**.

Reply:
I would propose to implement a refactorization of the compartmental model tests in a separate PR, as it is not really related to the topic of this PR specficially, and create an associated issue outlining all problems that need to be addressed.